modules/ossm-control-plane-configuration-migration-requirements.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+
+// * service-mesh-docs-main/migrating/cluster-wide/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-cluster-wide-migration_{context}"]
+= Control plane configuration migration requirements
+
+During the migration process, two cluster-wide control planes run in the same cluster while the data plane namespaces are gradually migrated to the {SMProductName} 3.0 installation. One control plane is associated with the {SMProductName} {SMv2Version} installation and the other is associated with the {SMProduct} 3.0 installation. You must carefully plan out the migration steps to avoid possible conflicts between the two control planes.
```

Comment:
```suggestion
During the migration process, two cluster-wide control planes run in the same cluster while the data plane namespaces are gradually migrated to the {SMProductName} 3.0 installation. One control plane is associated with the {SMProductName} {SMv2Version} installation and the other is associated with the {SMProduct} 3.0 installation. You must carefully plan the migration steps to avoid possible conflicts between the two control planes.
```
- Just a nit here. "Plan out" can be simplified to "plan".

Reply:
done